dts/bindings/serial/uart.yaml

```diff
@@ -24,6 +24,11 @@ properties:
       category: required
       description: Initial baud rate setting for UART
       generation: define
+    hardware-fc-rtscts:
```

Comment:
Name seems ok to me but I'd like @galak to comment on this.

Reply:
@Benichou34 , seems that a `hw-flow-control` is already available for the same purpose, can you check?

Reply:
I used the "hw-flow-control" as dts binding keyword.